.github/scripts/cygwin/execute-tests.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+source `dirname ${BASH_SOURCE[0]}`/../config.sh
+
+echo "::group::Execute Cygwin AArch64 tests"
+    pushd $ARTIFACT_PATH
+        set +e
+        # Current workarounds are not enough to catch output stream however it might be validated manually
+        powershell -Command ".\hello-cygwin.exe"
+        message=$(powershell -Command ".\hello-cygwin.exe; echo \$LASTEXITCODE")
```

Comment:
hello cygwin example should be fixed to be able to catch output, now it is testing only exit code

Reply:
fixed in aarch64-cygwin-wip-v3

Reply:
I the powersell command wrapper necesary then? Or WSL interop still do not deliver the output or the exit code?

Reply:
Yes, it works fine now.